src/Module.ts

```diff
@@ -298,6 +300,26 @@ export default class Module {
 			get dynamicImporters() {
 				return dynamicImporters.sort();
 			},
+			get exportedBindings() {
+				const exportBindings: Record<string, string[]> = { '.': [...exports.keys()] };
+
+				for (const [name, { source }] of reexportDescriptions) {
+					(exportBindings[source] ??= []).push(name);
```

Comment:
Ah, I am sorry but there is one last thing I just noticed. Currently, we are putting the relative imports into `exportedBindings`. I do not think this is really as useful as
- there are different ways to import the same module (use file extension? node-resolve directory import? something via node_modules?)
- it would require anyone relying on this to first resolve the import themselves

Instead, I think it would be more useful to use the unique module ids in this place. That would also be consistent with what you find in `importedBindings`. The change is easy, though: just use module.id instead

```suggestion
				for (const [name, { module: { id } }] of reexportDescriptions) {
					(exportBindings[source] ??= []).push(id);
```

Reply:
Yeah, unique module id is best, but sometimes the module maybe is `null`, e.g, access `info.exportedBindings` in `moduleParsed` hook. Btw, I noticed the `source` in `exportAllSources` is relative path.

Reply:
Ah I see, you are right. That information is only available after another resolution phase, at least it is not information contained in the module. Then I think it is fine.